assembly/src/library/masl.rs

```diff
@@ -281,7 +281,7 @@ impl Serializable for MaslLibrary {
             LibraryPath::strip_first(&module.path)
                 .expect("module path consists of a single component")
                 .write_into(target);
-            module.ast.write_into(target);
+            module.ast.write_into(target, AstSerdeOptions::new(false));
```

Comment:
Should we keep track of serialization options as a part of the `MaslLibrary` struct? (i.e., add it on line 28 above).

Reply:
We can, but it feels like a scope creep. 

Up to now `MaslLibrary` has never serialized imports, and this PR doesn't change that. The `AstSerdeOptions` used by `MaslLibrary` are therefore effectively a constant. I think we should change the `MaslLibrary` struct when we need to add serialization of imports to it.

That said, it would make sense to define the options as a constant for now, just so that it's clear what is happening.

Reply:
I think using a constant is fine - but two things:

1. I'd probably serialize imports by default - so, use `AstSerdeOptions::new(true)`.
2. I would still update `Module` serialization as I wrote in the comment below.